rosetta-source/src/main/rosetta/base-staticdata-party-enum.rosetta

```diff
@@ -81,7 +81,6 @@ enum PartyRoleEnum: <"The enumerated values for the party role. The enumerated v
     ClearingOrganization <"The organization that acts as a central counterparty to clear a derivatives contract.  This is used to represent the role of Central Counterparties (CCPs) or Derivative Clearing Organizations (DCOs).  Sometimes called 'ClearingService'. Some implementations also use the term 'Clearer'.">
     Client <"Client as defined under ESMA MIFIR. This is generally the investor or other client of an investment firm, and is synonymous with the Beneficiary in many circumstances.">
     ClientDecisionMaker <"The party or person who, having legal authority to act on behalf of a trade counterparty, made the decision to acquire or sell the financial instrument.">
-    CompressionServiceProvider <"The party that acts as a compression service provider, by compressing existing portfolios of trades to reduce notional amount outstanding, the number of transactions or otherwise harmonise the terms, by combining or offsetting trades and commonly to replace the terminated derivatives with new replacement trades.">
```

Comment:
If this goes against Prod, this shouldn't remove the enum value (backward incompatible) but instead should annotate it as `[deprecated]`.

You can fully remove the enum in a Pull Request against Dev. In this instance it should be documented as a *backward-incompatible change* in the release notes, with instructions for how to remap (from `CompressionServiceProvider` to `PTRRServiceProvider`), ideally with pointers to relevant samples.

Reply:
I have commited again. I have added the annotation [deprecated] as a comment because if not I was getting an error

Reply:
Good. Make sure you also raise another PR against Dev.